### ISSUE TITLE
sugestão: Suporte ao GUIGO.TV

### ISSUE THREAD

**Issue by alexetedans** [olavopeixoto/plugin.video.brplay#141]
Não sei se muita gente assina, mas gostaria se possível de ver o serviço no add-on pois o aplicativo deles é muito ruim, se precisar testar te envio os dados de acesso, mas no site também é possivel testar por 7 dias

**Comment by olavopeixoto** [comment]
Não pretendo implementar nenhum serviço de IPTV no addon. Somente fontes oficiais.

**Comment by alexetedans** [comment]
Concordo, mas o serviço que me mencionemos é pirata é homologado assim com a Oi tv ou Direct tv
